grizzly/environment.py

```diff
@@ -45,6 +45,30 @@ def after_feature(context: Context, feature: Feature, *args: Tuple[Any, ...], **
         if return_code != 0:
             feature.set_status('failed')
 
+        grizzly = cast(GrizzlyContext, context.grizzly)
+        stats = grizzly.state.environment.stats
+
+        for behave_scenario in cast(List[Scenario], feature.scenarios):
+            grizzly_scenario = grizzly.scenarios.find_by_description(behave_scenario.name)
+            if grizzly_scenario is None:
+                continue
+
+            scenario_stat = stats.get(f'{grizzly_scenario.identifier} {grizzly_scenario.description}', 'SCEN')
```

Comment:
As 'SCEN' is used hardcoded in >1 files, maybe it would be nice to introduce some type/constant for the possible values to improve maintainability?